src/Skipper/Contract/SkipVoterInterface.php

```diff
@@ -4,9 +4,11 @@
 
 namespace Rector\Skipper\Contract;
 
+use PhpParser\Node;
+
 interface SkipVoterInterface
 {
     public function match(string | object $element): bool;
 
-    public function shouldSkip(string | object $element, string $filePath): bool;
+    public function shouldSkip(string | object $element, string $filePath, ?Node $node): bool;
```

Comment:
this is bc break, probably use service on `Skipper::shouldSkipCurrentNode()` instead:

https://github.com/rectorphp/rector-src/blob/a22c3b9ff9f1889902ec42ccd3f4e3c15ac42a4b/src/Skipper/Skipper/Skipper.php#L60